### PR TITLE
Added a note on Insights query API

### PR DESCRIPTION
**Pull request by SudheendraKatte**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [x] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*
[DOC-2255](https://harness.atlassian.net/browse/DOC-2255)

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot. 

[DOC-2255]: https://harness.atlassian.net/browse/DOC-2255?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/first-gen/continuous-delivery/continuous-verification/new-relic-verification/1-new-relic-connection-setup.md [hunk 1]
```diff
@@ -28,16 +35,16 @@ To connect a verification provider, do the following:
    
 4. Complete the following fields of the **Add New Relic Verification Provider** dialog.
 
-### Step 2: Encrypted API Key
+### Step 2: Encrypted API key
 
-For secrets and other sensitive settings, select or create a new [Harness Encrypted Text secret](../../../firstgen-platform/security/secrets-management/use-encrypted-text-secrets.md).Enter the API key needed to connect with the server. For information on API keys, see [Access to REST API keys](https://docs.newrelic.com/docs/apis/getting-started/intro-apis/access-rest-api-keys) from New Relic.
+For secrets and other sensitive settings, select or create a new [Harness Encrypted Text secret](../../../firstgen-platform/security/secrets-management/use-encrypted-text-secrets.md).Enter the API key needed to connect with the server. To learn more about API keys, go to [Access to REST API keys](https://docs.newrelic.com/docs/apis/getting-started/intro-apis/access-rest-api-keys).
```
Comment (michaelcretzman) [inline docs/first-gen/continuous-delivery/continuous-verification/new-relic-verification/1-new-relic-connection-setup.md:40]:
```suggestion
For secrets and other sensitive settings, select or create a new [Harness Encrypted Text secret](../../../firstgen-platform/security/secrets-management/use-encrypted-text-secrets.md). Enter the API key needed to connect with the server. To learn more about API keys, go to [Access to REST API keys](https://docs.newrelic.com/docs/apis/getting-started/intro-apis/access-rest-api-keys).
```

Comment (michaelcretzman) [reply docs/first-gen/continuous-delivery/continuous-verification/new-relic-verification/1-new-relic-connection-setup.md:40]: needed a space after a period. might have happened during migration. see a lot of these in the docs now.

**Review by michaelcretzman — APPROVED** [review]
approved with one minor suggestion.

**Comment by SudheendraKatte** [comment]
Incorporated review comments by @michaelcretzman

**Pull request merged**